### ISSUE TITLE
Mixed tabs and spaces

### ISSUE THREAD

**Issue by taneltm** [abcdan/together#2]
Looks like `app/src/routes/blog/_posts.js` has mixed tabs and spaces.
Looks like that might have been intentional, but usually when you configure the editor to use spaces for indentation, you can’t make the tab symbol anymore.

**Comment by abcdan** [comment]
Yep, I've noticed this as well. I'm pretty sure this is from the default template we work off.

I'll look at all files asap and fix it.

**Comment by taneltm** [comment]
Also found that when you use tabs for indentation, there will be WAR!  
But GitHub will also look at the `.editorconfig` file in the project root and will configure tab width based on what is set there.

https://stackoverflow.com/a/33831598
